societe-archeologique-de-bordeaux.csl

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="never" default-locale="fr-FR" page-range-format="expanded">
+  <info>
+    <title>Société Archéologique de Bordeaux</title>
+	    <title-short>SAB</title-short>
+    <id>http://www.zotero.org/styles/societe-archeologique-de-bordeaux</id>
+    <link href="http://www.zotero.org/styles/societe-archeologique-de-bordeaux" rel="self"/>
```

Comment:
(see also https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link for more context)